reliance_edge/include/redconf.h

```diff
@@ -51,7 +51,7 @@
 
 #define REDCONF_API_FSE_TRANSMASKSET 0
 
-#define REDCONF_OUTPUT 1
+#define REDCONF_OUTPUT 0
```

Comment:
What will this change?

Reply:
The output wasn't actually going anywhere but it was still being made. This is a performance improvement